Content/Items/Breacher/Projectiles.BreachCannonSentry.cs

```diff
@@ -0,0 +1,585 @@
+				sb.Draw(ballTex, superLaserEndpoint - Main.screenPosition, null, Color.White, superRotation, ballTex.Size() / 2, scale * 0.85f * superLaserSizeMult, SpriteEffects.None, 0f);
+			}
+
+
+			Main.spriteBatch.End();
+			Main.spriteBatch.Begin(default, endState, default, default, default, default, Main.GameViewMatrix.TransformationMatrix);
+		}
+
+		public void DrawAdditive(SpriteBatch sb)
+		{
+			Color blue = new Color(0,0,255);
+			Color blueCyan = Color.Lerp(Color.Cyan, blue, 0.5f);
+			var tex = ModContent.Request<Texture2D>(AssetDirectory.Assets + "Keys/GlowSoft").Value;
+
+			sb.Draw(tex, laserStartpoint - Main.screenPosition, null, blueCyan, 0, tex.Size() / 2, 0.45f * laserSizeMult, SpriteEffects.None, 0f);
+			sb.Draw(tex, laserEndpoint - Main.screenPosition, null, blueCyan, 0, tex.Size() / 2, 0.35f * laserSizeMult, SpriteEffects.None, 0f);
+
+			if (superLaser)
+			{
+				sb.Draw(tex, superLaserStartpoint - Main.screenPosition, null, blueCyan, 0, tex.Size() / 2, 0.45f * superLaserSizeMult, SpriteEffects.None, 0f);
+				sb.Draw(tex, superLaserEndpoint - Main.screenPosition, null, blueCyan, 0, tex.Size() / 2, 0.35f * superLaserSizeMult, SpriteEffects.None, 0f);
+			}
+
+			DrawBalls(sb, BlendState.Additive, Color.White, Color.White, 0.35f);
+		}
+
+		private void SpawnParticles()
+		{
+			Vector2 direction = laserEndpoint.DirectionTo(laserStartpoint);
+
+			Color color1 = Color.Cyan;
+			Color color2 = new Color(0, 0, 255);
+
+			for (int i = 0; i < 5; i++)
+				Dust.NewDustPerfect(laserEndpoint, ModContent.DustType<BreachImpactGlow>(), direction.RotatedByRandom(0.6f) * Main.rand.NextFloat(12), 0, Color.Lerp(color1, color2, Main.rand.NextFloat()), Main.rand.NextFloat(0.25f, 0.6f));
+
+			for (int i = 0; i < 3; i++)
+			{
+				Vector2 vel = direction.RotatedByRandom(0.6f) * Main.rand.NextFloat(9);
+				Dust.NewDustPerfect(laserEndpoint + new Vector2(0, 35) + (vel * 2), ModContent.DustType<BreachImpactSpark>(), vel, 0, Color.Lerp(color1, color2, Main.rand.NextFloat()), Main.rand.NextFloat(1.25f, 1.6f));
+			}
+
+			if (superLaser)
+            {
+				direction = superLaserEndpoint.DirectionTo(superLaserStartpoint);
+
+				for (int i = 0; i < 5; i++)
+					Dust.NewDustPerfect(superLaserEndpoint, ModContent.DustType<BreachImpactGlow>(), direction.RotatedByRandom(0.6f) * Main.rand.NextFloat(12), 0, Color.Lerp(color1, color2, Main.rand.NextFloat()), Main.rand.NextFloat(0.25f, 0.6f));
+
+				for (int i = 0; i < 3; i++)
+				{
+					Vector2 vel = direction.RotatedByRandom(0.6f) * Main.rand.NextFloat(9);
+					Dust.NewDustPerfect(superLaserEndpoint + new Vector2(0, 35) + (vel * 2), ModContent.DustType<BreachImpactSpark>(), vel, 0, Color.Lerp(color1, color2, Main.rand.NextFloat()), Main.rand.NextFloat(1.25f, 1.6f));
+				}
+			}
+		}
+	}
+	public class BreachImpactGlow : Dusts.Glow
```

Comment:
missing linebreak here, it may also be wise to seperate these dusts out to their own file in the dusts directory